src/renderer/contexts/SettingsContext.tsx

```diff
@@ -42,21 +35,7 @@ interface Settings {
 export const SettingsContext = createContext<Readonly<Settings>>({} as Settings);
 
 export const SettingsProvider = memo(({ children }: React.PropsWithChildren<unknown>) => {
-    // Global Settings
-    const useIsCpu = useMemoArray(useLocalStorage('is-cpu', false));
-    const useIsFp16 = useMemoArray(useLocalStorage('is-fp16', false));
-    const usePyTorchGPU = useMemoArray(useLocalStorage('pytorch-gpu', 0));
-    const useNcnnGPU = useMemoArray(useLocalStorage('ncnn-gpu', 0));
-    const useOnnxGPU = useMemoArray(useLocalStorage('onnx-gpu', 0));
-    const useOnnxExecutionProvider = useMemoArray(
-        useLocalStorage('onnx-execution-provider', 'CUDAExecutionProvider')
-    );
-    const useOnnxShouldTensorRtCache = useMemoArray(
-        useLocalStorage('onnx-should-tensorrt-cache', false)
-    );
-    const useOnnxShouldTensorRtFp16 = useMemoArray(
-        useLocalStorage('onnx-should-tensorrt-fp16', false)
-    );
+    const useBackendSettings = useMemoArray(useLocalStorage('backend-settings', {}));
```

Comment:
Settings migrations?

Reply:
I was just going to announce that python related settings are being reset, but yeah I guess I could try adding a migration for settings

Reply:
Man, how would i even go about migrating this?

Reply:
Yeah, then let's not do this.